rsocket-core/src/main/java/io/rsocket/DuplexConnection.java

```diff
@@ -86,6 +87,17 @@
    */
   ByteBufAllocator alloc();
 
+  /**
+   * Return the remote address that this connection is connected to. The returned {@link
+   * SocketAddress} varies by transport type and should be downcast to obtain more detailed
+   * information. For TCP and WebSocket, the address type is {@link java.net.InetSocketAddress}. For
+   * local transport, it is {@link io.rsocket.transport.local.LocalSocketAddress}.
```

Comment:
Can we clarify here that it is null for disconnected connections? Do we want to mark as @Nullable?

Reply:
I think for disconnected, the channel still returns the address. It can return null before it is connected is my understanding but that the `DuplexConnection` is created only after a connection is ready.

Reply:
Yup. I think soo. I guess once the connection is resolved, the address should be available.